From d03b6ae69b59beea85e77a778a61b6ede77c2f64 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Mon, 5 Jan 2015 14:10:42 +0100
Subject: [PATCH 49/99] HACK HACK HACK: Don't use mingw format strings

This should be removed when our toolchain isn't considered as vanilla mingw
---
 include/vlc_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/vlc_common.h b/include/vlc_common.h
index 51bf9cd..0c00ec8 100644
--- a/include/vlc_common.h
+++ b/include/vlc_common.h
@@ -64,7 +64,7 @@
 #endif
 
 /* Try to fix format strings for all versions of mingw and mingw64 */
-#if defined( _WIN32 ) && defined( __USE_MINGW_ANSI_STDIO )
+#if defined( _WIN32 ) && defined( __USE_MINGW_ANSI_STDIO ) && !defined( _MSC_VER )
  #undef PRId64
  #define PRId64 "lld"
  #undef PRIi64
-- 
1.9.5.msysgit.0

